pkg/common/cluster_actions.go

```diff
@@ -257,7 +257,7 @@ func (i *ClusterActionRunner) configureBrowserRedirector(provider, flow string,
 	if authenticatorConfig == nil && flow != "" {
```

Comment:
Should this conditional also be checking for `provider` instead of `flow`?

Reply:
yes, makes sense